docs/design/jobflow/README.md

```diff
@@ -289,7 +299,19 @@ Here is an example of jobflow:
 * The difference between jobtemplate and vcjob is that jobtemplate will not be issued by the job controller, and jobflow can directly reference the name of the JobTemplate to implement the issuance of vcjob.
 * JobFlow supports making changes to jobtemplate when referencing jobtemplate
 
-#### Definition
+####action of jobtemplate and response impact
+
+**create jobtemplate**:
+
+Create a jobtemplate to be used by jobflow.
+
+**update jobtemplate**:
+
+After the jobtemplate is updated, it will not affect the vcjobs that have been created based on the jobtemplate. It will not affect the successfully executed jobflow. It may affect the jobflow that has not been executed. For example, the jobflow that has not been executed to the jobtemplate stage will use the updated jobtemplate template.
+
+**delete jobtemplate**:
+
+After the jobtemplate is deleted, it will not affect the vcjob that was originally created based on the jobtemplate. It will not affect the successfully executed jobflow. It may affect the jobflow that has not been executed. For example, the jobflow that has not been executed to the jobtemplate stage will not find the jobtemplate template and cannot proceed to the next step.
```

Comment:
Something should be done here, such as:
It is not allowed to delete the jobtemplate when there is jobflow using this jobtemplate

Reply:
I agree with this comment, I will add this judgment in the webhook module